core/lib/Cardano/Address/Script.hs

```diff
@@ -255,6 +286,10 @@ validateScript = \case
         when (length script < fromIntegral m) $ Left ListTooSmall
         when (hasDuplicate script) $ Left DuplicateSignatures
         traverse_ validateScript script
+
+    ValidFromSlot _ -> pure ()
```

Comment:
Is `0` an accepted value though :thinking: ? (I mean, purely from the protocol perspective, not in the context of a particular script).

Reply:
not really -> https://github.com/input-output-hk/cardano-ledger-specs/blob/79055d5d482fb02c86f8b712eb4940f6e3c25a69/shelley-ma/impl/src/Cardano/Ledger/ShelleyMA/Timelocks.hs#L233

I plan to add someday time interval based validation - based on Allen interval algebra.